### ISSUE TITLE
buildWatch functions is still choosing the jspm_packages/npm

### ISSUE THREAD

**Issue by Herman-Scheer** [jspm/jspm-cli#1820]
I noticed there was an issue (#1737) already opened with a merged fix. However the problem still persist under a file structure where jspm_packages/npm length and the application folder structure length are the same (7) I had to incorporate this hack in order to get the project to watch my project files.

node_modules/jspm/lib/bundle.js
```
    var lowestDir = path.dirname(files[0]);
    files.forEach(function(file) {
      if (path.dirname(file).split('/').length < lowestDir.split('/').length && path.dirname(file).split('/').indexOf('jspm_packages') < 0) {
        lowestDir = path.dirname(file);
      }
    });
```

I'm basically ignoring jspm_packages from the buildWatch which might not be the best option.

I set up an example project to show the error: https://github.com/Herman-Scheer/testing-jspm

just npm instal and jspm install and the npm run bundle.

The console will log watching jspm_packages/npm

 Would love to get some guidance @guybedford as to how I can make a reasonable pull request, or if there is something already on the way. Thank you.

**Comment by guybedford** [comment]
I did notice this and I believe this is fixed on the 0.17 branch at https://github.com/jspm/jspm-cli/blob/0.17/lib/bundle.js#L275. The next release should be this week.

**Comment by guybedford** [comment]
The latest release is out now, so this should be working. If not please post back here.
